pkg/cmd/pulumi/operations/destroy.go

```diff
@@ -352,12 +357,20 @@ func NewDestroyCmd() *cobra.Command {
 		"Specify a single resource URN to destroy. All resources necessary to destroy this target will also be destroyed."+
 			" Multiple resources can be specified using: --target urn1 --target urn2."+
 			" Wildcards (*, **) are also supported")
+	excludes = cmd.PersistentFlags().StringArrayP(
+		"exclude", "x", []string{},
```

Comment:
I'm kinda with Thomas on this -- I think we can leave out a short option for now and just have `--exclude` and see if people want/need it in the future.